app/src/main/java/org/exoplatform/activity/BoardingActivity.java

```diff
@@ -223,4 +295,47 @@ public void run() {
     public void statusBarColor(){
         getWindow().setStatusBarColor(getResources().getColor(R.color.status_bar_color,this.getTheme()));
     }
+
+    public static boolean checkVersionUpdate(String olderVerison, String newVersion) {
+        if (olderVerison.length() == 0 || newVersion.length() == 0) {
```

Comment:
Code seems to be complicated, if this will just check version from Playstore, you could simply remove the dots characters from version and then compare both Integers.